Sample which showcases how to connect traces and logs with OpenTelemetry (#5750)
Particular/docs.particular.net: samples/open-telemetry/logging/sample.md

---
title: Connecting OpenTelemetry traces and logs
summary: How to connect OpenTelemetry traces to logs
reviewed: 2022-06-27
component: Extensions.Logging
related:
- nservicebus/logging
---

This sample shows how to configure an NServiceBus endpoint to collect telemetry data and link the traces and logs together.

> First, OpenTelemetry needs to be configured on the endpoint:

Reviewer: Something about this felt off. I'm not happy with the suggestion either

```suggestion
First, OpenTelemetry needs to be configured to capture NServiceBus traces:
```
